aiohttp/helpers.py

```diff
@@ -60,7 +61,7 @@
 else:
     import async_timeout
 
-__all__ = ("BasicAuth", "ChainMapProxy", "ETag")
+__all__ = ("BasicAuth", "ChainMapProxy", "ETag", "reify")
```

Comment:
Does this really need to go into `__all__`? Do we expect the end-users to do `from aiohttp.helpers import *` and get it in their env as opposed to importing directly from the lib?

Reply:
I've [seen issues where its used](https://github.com/aio-libs/aiohttp/issues/7993#issuecomment-2327223933).. but even then its not import at that point so I think we can take this out